### ISSUE TITLE
wp_cli with --show-bulk-errors doesn't display errors

### ISSUE THREAD

**Issue by sushicodeur** [10up/ElasticPress#729]
When I run 

`wp elasticpress index --setup --show-bulk-errors`

The output shows :

```
- 123: Post title #1
	Array
- 234: Post title #2
	Array
- 567: Post title #3
	Array
```

I would have hoped a more verbose output.

**Comment by ivankristianto** [comment]
Can you try with this one:
`wp elasticpress index --setup --debug`

**Comment by tlovett1** [comment]
This should have been fixed in 2.2

**Comment by sushicodeur** [comment]
Thank you for your answers.

@tlovett1 yes I sed the plugin from master instead of the wordpress repository, it works fine, will wait for 2.2. Thank you !
